### DISCUSSION TITLE
Any recommendation for the propeller parameters between section r/R=0 and r/R=0.2 (e.g. first aerodynamic section) if there is no description given from the manufacturere like APC?

### DISCUSSION THREAD

**Discussion by ilm71** [byuflowlab/FLOWUnsteady#106] [General]
Hi, 
my question is linked to the headline - propeller manufacturer like APC are not giving detailed description on the propeller parameters from r/R zero to the first aerodynamic section - do you have ane best practise or recommendation which parameters for the first section should be given witin FlowUnsteady to avoid unrealistic thrust and torque results? 

Many Thanks

**Comment by Turbulence1116** [comment]
Hi, in the database, there are predefined propellers, the APC propellers are there too, you can use them directly. The relevant info can be found here: https://flow.byu.edu/FLOWUnsteady/examples/propeller-J040/

**Reply by ilm71** [reply]
Thanks, I have already checked that - my question war more related to a general approach (e.g. for different APC propellers), where I wanted to know if there is a best practice for the specific blade section.

For example: The geometry given for an APC 12x8 propeller starts at r/R=0.2 --> so the information for the section 0 to 0.2 is missing.

**Reply by Turbulence1116** [reply]
I think the missing part is accounted by the propeller hub definition, as different propeller has different hub. To get better prediction, a hub correction can be applied, see the end of here: https://flow.byu.edu/FLOWUnsteady/examples/rotorhover-aero/

**Reply by sakurakikiki** [reply]
r/R，R=Rtip？orR=Rtip-Rhub?

**Comment by EdoAlvarezR** [comment]
Hi Philipp, could you provide more context to your question?

In my experience, having an "open core" in the absence of a hub in the simulations affect the results very little as long as the hub correction is bringing the loading down to zero at the root, as @Turbulence1116 pointed out.

In a more general sense (not in the context of simulation best practices, but in design best practices), the section near the root produces very little thrust, while still contributing to drag (and torque). For this reason (plus structural reasons), you typically use a thicker airfoil to avoid separation at this slower local velocity, while reducing chord length. You would usually want to start transitioning to the root section at around r/R ~ 0.15-0.20 or even 0.25. For a variable-pitch blade you typically will want to transition into a thin cylinder at the root to hook up into the variable pitch mechanism.

**Comment by ilm71** [comment]
Hi All , 

many thanks for your recommendations - I played  a little bit with the parameters on the geometry side as well as on the hub/tip-loss-correction side and found for the geometry a good compromise in terms of thrust and torque prediction. After some iterations it was possible to match the experimental and numerical results accurately. - Thanky you very much again!

**Reply by Liuhaosheng1** [reply]
Hello, I would like to know how you specifically modified the tip/hub loss correction, and what specific methods you have to correct it based on certain data of the propeller.

**Reply by FabianaRugg** [reply]
i m also intrested in the changes applied
